digautoprofiler/profile_lib.py

```diff
@@ -282,7 +272,7 @@ def getTempAggrData(dfName: pd.DataFrame, tempColName: str, quantColName: str, a
     offsetAliases = ['Y','M','W','D','H','T','S']
     i = 0
     previousLen = -float("inf")
-    currentLen = 0
+    currentLen = len(dfName[tempColName].dt.to_period(offsetAliases[i]).unique())
```

Comment:
What is this doing?

Reply:
I find the initial value of `currentLen` is wrong, which is the reason why the temporal chart sometimes can not determine the number of bins well